internal/utils.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+const (
+	ArgoCDSecretTypeLabel   = "argocd.argoproj.io/secret-type"
+	ArgoCDSecretTypeCluster = "cluster"
+	AppSetKind              = "ApplicationSet"
+	AppSetAPIGroup          = "argoproj.io/v1alpha1"
+)
+
+func IsArgoCDCluster(labels map[string]string) bool {
+	val, ok := labels[ArgoCDSecretTypeLabel]
+	if val == ArgoCDSecretTypeCluster && ok {
```

Comment:
perhaps just return the condition without the if/else?

Reply:
Ah, good point. Fixed